src/app/components/Pokemon/PokemonInfo.tsx

```diff
@@ -0,0 +1,17 @@
+interface PokemonInfoProps {
+  name: string;
+  children: React.ReactNode;
```

Comment:
Should do:
Je t'encourage à ne plus définir toi même le type de `children` mais d'utiliser plutôt `React.PropsWithChildren` type.
ex: 
```jsx
export const Component = (props: PropsWithChildren<ComponentProps>) => 
```